presto/dataset.py

```diff
@@ -169,13 +170,14 @@ class WorldCerealLabelledDataset(WorldCerealBase):
     def __init__(
         self,
         dataframe: pd.DataFrame,
-        aezs_to_remove: Optional[List[int]] = None,
+        countries_to_remove: Optional[List[str]] = None,
         years_to_remove: Optional[List[int]] = None,
     ):
         dataframe = dataframe.loc[~dataframe.LANDCOVER_LABEL.isin(self.FILTER_LABELS)]
 
-        if aezs_to_remove is not None:
-            dataframe = dataframe[(~dataframe.aez_zoneid.isin(aezs_to_remove))]
+        if countries_to_remove is not None:
+            dataframe = self.join_with_world_df(dataframe)
+            dataframe = dataframe[(~dataframe.name.isin(countries_to_remove))]
```

Comment:
Are we somehow checking whether the exact string corresponds to the country name in the `world_df`? I'm afraid that at this point if we'd pass `"latvia"`, it would silently not filter out Latvian points because it doesn't exactly match with `"Latvia"`.
If this is the case, we should maybe do one or both of:
- putting everything here lower case before matching to already remove the case sensitivity
- check if the countries in `countries_to_remove` all have a hit in the `world_df`

Reply:
this is a good idea. I've [added the check](https://github.com/WorldCereal/presto-worldcereal/pull/43/commits/1d2aa84809d171188cff24795423a966b7212dae) to start.